examples/offix/test/offline.test.ts

```diff
@@ -43,18 +43,6 @@ async function newClient(
 // TODO: remove this queries once we expose client queries/mutations
 // TODO: from the TestxServer API
 // TODO: https://github.com/aerogear/graphql-testx/issues/15
-export const FIND_ALL_TASKS = gql`
-  query findAllTasks {
-    findAllTasks {
-      id
-      version
-      title
-      description
-      author
-    }
-  }
-`;
-
 export const ADD_TASK = gql`
```

Comment:
@vinifarias can't we substitute also the ADD_TASK mutation?

Reply:
@b1zzu we can't, because of [this](https://github.com/aerogear/graphback/issues/474) error. The fix has not been released yet.

Reply:
There is dev2 that has the fix. Working on releasing dev3

Reply:
Nice @wtrocki, thanks

Reply:
We will address this issue with the upgrade to 0.10